browser/ui/webui/settings/brave_privacy_handler.cc

```diff
@@ -133,19 +133,27 @@ void BravePrivacyHandler::OnStatsUsagePingEnabledChanged() {
   }
 }
 
-#if BUILDFLAG(BRAVE_P3A_ENABLED)
 void BravePrivacyHandler::SetP3AEnabled(const base::Value::List& args) {
-  SetLocalStateBooleanEnabled(brave::kP3AEnabled, args);
+#if defined(OFFICIAL_BUILD)
+  SetLocalStateBooleanEnabled(p3a::kP3AEnabled, args);
+#endif
 }
 
 void BravePrivacyHandler::GetP3AEnabled(const base::Value::List& args) {
-  GetLocalStateBooleanEnabled(brave::kP3AEnabled, args);
+#if defined(OFFICIAL_BUILD)
+  GetLocalStateBooleanEnabled(p3a::kP3AEnabled, args);
+#else
+  CHECK_EQ(args.size(), 1U);
+  AllowJavascript();
+  ResolveJavascriptCallback(args[0], base::Value(false));
+#endif
 }
 
+#if defined(OFFICIAL_BUILD)
```

Comment:
we need to miminise using it as disccused in DM

Reply:
removed